yarn.lock

```diff
@@ -0,0 +1,3271 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Nope! One of the advantages of yarn is the lock file ensures we all use the exact same dependencies, designed to be checked in https://yarnpkg.com/en/docs/yarn-lock